test/Microsoft.ML.CodeAnalyzer.Tests/Code/InstanceInitializerTest.cs

```diff
@@ -23,22 +27,19 @@ class TypeName
         private static int _muck = 4;
         private readonly float _blorg = 3.0f;
         private string _fooBacking;
-        public string Foo { get; set => _fooBacking = value; }
+        public string Foo { get => _fooBacking; set => _fooBacking = value; }
```

Comment:
📝 I went ahead and fixed this compilation error in a test input. It seemed unintentional.